elixir/apps/api/lib/api/gateway/views/subject.ex

```diff
@@ -0,0 +1,10 @@
+defmodule API.Gateway.Views.Subject do
+  alias Domain.Auth
+
+  def render(%Auth.Subject{} = subject) do
+    %{
+      name: subject.actor.name,
+      email: subject.identity.email
```

Comment:
`identity` is not a required field on subject, so this will raise a runtime error for headless clients. Service Accounts and API clients don't have identities associated with them.

In my PR (#10505) I improve things here by moving `email` to actor, so what you may want to do is just use `client.actor.email` here and then park this until that PR goes in, which should hopefully be this week.

```suggestion
      email: client.actor.email
```

Reply:
Also `name` will be moving to `identity` (along with `given_name` `surname` etc).

Reply:
`client.actor` is already there as well, so you could pull the email from that.